src/tests/api/Filter.test.js

```diff
@@ -0,0 +1,136 @@
+import { applyFilter } from 'src/api';
+import { filters } from './data-mocks/filters';
+
+const makePossibleFeature = (value) => [
+  {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [0, 0]
+    },
+    properties: {
+      column1: value
+    }
+  },
+  {
+    column1: value
+  }
+];
+
+describe('Filters', () => {
+  test('should return 1 if no filters present', () => {
+    const params = { filters: {}, type: 'number' };
+    expect(applyFilter(params)()).toBe(1);
```

Comment:
Probably obvious, but why do we get a 'number' here as result (1) instead of true / false?

Reply:
That's because at the end, deck.gl [`filterRange` prop is set to `[1, 1]`](https://github.com/CartoDB/carto-react-lib/blob/a80ec3efca842b5c7bc683cc763277d316e86d27/src/api/useCartoLayerFilterProps.js#L15).